src/lintlist.rs

```diff
@@ -0,0 +1,2274 @@
+/// Lint data parsed from the Clippy source code.
```

Comment:
I think it would be good to have a module level comment here as well that clarifies that this file is auto-generated when running `util/dev update_lints`